src/main/java/ticTacToe/player/SmartComputer.java

```diff
@@ -25,6 +25,7 @@ public String makeMove(Ui ui, Grid grid, Lines lines) {
         } else {
             move.append(bestMovePosition(grid, lines));
         }
+        slowDown();
```

Comment:
It looks like the SmartComputer moves are always slowed down now, even with HvC. I don't think that should be happening, I think the delay should only be for CvC.

Reply:
I see... I didn't think that might be a problem

Reply:
I tried to fix this. Also fixed the too-slowly-running-tests issue (see last commit below after `New changes since you last viewed` (is this a new feature??)